ci: select_tests fail during cron runs since last merge

[Original merge run](https://travis-ci.org/openSUSE/openSUSE-release-tools/jobs/505696061) passed, but subsequent [cron runs](https://travis-ci.org/openSUSE/openSUSE-release-tools/jobs/506565254) fail.

```
FAIL: test_select_comments (tests.select_tests.TestSelect)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/openSUSE/openSUSE-release-tools/tests/select_tests.py", line 48, in test_select_comments
    self.assertEqual(len(second_select_comments) - 1, len(first_select_comments))
AssertionError: 0 != 1
```

--- a/osclib/stagingapi.py
+++ b/osclib/stagingapi.py
@@ -737,7 +737,7 @@ class StagingAPI(object):
         description = root.find('description')
         # Order the requests and replace it with yaml
         meta['requests'] = sorted(meta.get('requests', []), key=lambda x: x['id'])
-        yaml_new = yaml.dump(meta)
+        yaml_new = yaml.dump(meta, default_flow_style=None)
         if yaml_new == description.text:
             return
         description.text = yaml_new

